crates/wasi-http/tests/all/http_server.rs

```diff
@@ -107,7 +115,7 @@ impl Server {
 
 impl Drop for Server {
     fn drop(&mut self) {
-        tracing::debug!("shutting down http1 server");
+        debug!("shutting down http1 server");
         // Force a connection to happen in case one hasn't happened already.
         let _ = TcpStream::connect(&self.addr);
```

Comment:
I think this may need to be done `self.conn` times? (or thread that through here perhaps)